tests/fixtures/cli.py

```diff
@@ -39,5 +39,5 @@ def project_files_cli(test_dir, compatible_copy_tree):
     # clean-up
     Project.deactivate()
     os.chdir(test_dir)
-    shutil.rmtree(project.root)
+    # shutil.rmtree(project.root)
```

Comment:
@visch do you know if these files would linger indefinitely in a dev's system when running tests locally?